bundle/pom.xml

```diff
@@ -59,6 +59,7 @@
                             com.day.cq.wcm.workflow.process;version="[6.0,8)", <!-- using a wider version range for forward compatibility -->
                             com.day.cq.mailer;version="[5.9,7)", <!-- using a wider version range for forward compatibility -->
                             com.adobe.cq.sightly;version="[2.5,4)",
+                            javax.annotation;resolution:=optional, <!-- pulled by the tika-parsers dependency -->
```

Comment:
Hmmm. Why does adding a dependency to tika-parsers require a change to the Import-Package configuration here? That doesn't sound right. tika isn't embedded, right?

Reply:
Ah, I see the problem now. In 6.2 and 6.3, this package is exported by the system bundle with a minimal version (`0.0.0.1_008_JavaSE` or something like that). Currently, ACS AEM Commons imports this package with no version (so an effective version of 0 to infinity because we get it from a non-OSGI jar. When Tika brings in `javax.annotation:javax.annotation-api`, that jar has OSGi metadata which causes the range to be `[1.2,2)`.

Rather than using `exclusions` I think the correct approach would be to set the scope as `provided`. This should be done for all dependencies, but it looks like @badvision forgot to do this for POI. I'll fix that in master.